test/api-helper/project/withProject.test.ts

```diff
@@ -36,13 +37,15 @@ describe("withProject", () => {
         it("should call action on cloned project", async () => {
             const fgi = fakeGoalInvocation(GitHubRepoRef.from({ owner: "atomist", repo: "sdm" }), {
                 projectLoader: {
-                    doWithProject<T>(params: ProjectLoadingParameters, action2: WithLoadedProject<T>): Promise<T> {
+                    doWithProject<T>(params: ProjectLoadingParameters, wlp: WithLoadedProject<T>): Promise<T> {
                         const p = InMemoryProject.of(new InMemoryProjectFile("foo", "")) as any;
-                        return action2(p);
+                        return wlp(p);
                     },
                 },
             } as any);
-            const action = async (gi: GoalInvocation & ProjectListenerInvocation) => {
+            const action = async (gi: ProjectAwareGoalInvocation) => {
+                const r = await gi.exec("pwd");
```

Comment:
`pwd` is not available on MS Windows so this test will fail. It's best to call `node` if you want to actually run a command in tests.